pkg/label/label.go

```diff
@@ -0,0 +1,14 @@
+package label
+
+const (
+	// Cluster label is a new style label for ClusterID
+	Cluster = "giantswarm.io/cluster"
+	// ManagedBy label denotes which operator manages corresponding resource.
+	ManagedBy = "giantswarm.io/managed-by"
```

Comment:
Can we pull these two from https://github.com/giantswarm/k8smetadata/tree/main/pkg/label ?

Reply:
yeah good shout